### DISCUSSION TITLE
Is there a way to generate workflows without Krita GUI?

### DISCUSSION THREAD

**Discussion by uros-mijajlovic** [Acly/krita-ai-diffusion#1016] [General]
Is there a way to generate the same comfyui workflows that ai-diffusion plugin generates but without using the Krita GUI? For the input I would provide the image, the mask, and the prompt, and as an output i would get workflow.json

**Comment by Acly** [comment]
There is a "Dump Workflow" option in Interface settings. It writes the workflow.json for your last action.
